view/templates/_macros.twig

```diff
@@ -56,3 +56,11 @@
         </ul>
     </li>
 {% endmacro %}
+
+{% macro sitemap(page) %}
+    {% from _self import sitemap %}
+    <url><loc>https://docs.bolt.cm{{ page.url }}</loc></url>
+    {% for sub in page.children if sub is not empty %}
```

Comment:
The `if` isn't needed here. And would change to `page.subPages` with menu change.